activesupport/lib/active_support/testing/assertions.rb

```diff
@@ -5,6 +5,50 @@ module Testing
     module Assertions
       UNTRACKED = Object.new # :nodoc:
 
+      # Asserts that a message matching <tt>regex</tt> was logged
+      # within the provided block. Passes if content logged in
+      # the block matches the regex provided.
+      #
+      #   assert_logs /an informational log message/ do
+      #     Rails.logger.info 'an informational log message'
+      #   end
+      def assert_logs(regex)
+        old_logger = Rails.logger
+        log = StringIO.new
+        Rails.logger = Logger.new(log)
+
+        begin
+          yield
+
+          log.rewind
+          assert_match message, log.read, message
+        ensure
+          Rails.logger = old_logger
+        end
+      end
+
+      # Asserts that no messages matching the provided regex
+      # were logged in the provided block. Passes if the content
+      # logged inside the block doesn't match the regex provided.
+      #
+      #   assert_no_logs /an informational log message/ do
+      #     Rails.logger.info 'not very informational'
+      #   end
+      def assert_no_logs(regex)
+        old_logger = Rails.logger
+        log = StringIO.new
+        Rails.logger = Logger.new(log)
+
+        begin
+          yield
+
+          log.rewind
+          assert_no_match message, log.read
```

Comment:
Ditto: you intend to pass `regex` here. If `assert_logs` accepts a message parameter, so should `assert_no_logs`.